build.sh

```diff
@@ -39,6 +39,7 @@ HELP="$0 [clean] [libcudf] [cudf] [cudfjar] [dask_cudf] [benchmarks] [tests] [li
    --opensource_nvcomp           - disable use of proprietary nvcomp extensions
    --show_depr_warn              - show cmake deprecation warnings
    --ptds                        - enable per-thread default stream
+   --disable_large_strings       - disable large strings support
```

Comment:
nit: cmake build default options has large string disabled. build.sh without args, makes large string enabled. is there a reason for this difference ?

Reply:
FYI this will change the default in our conda packages, which use defaults from build.sh.

Reply:
I would prefer that by default cmake builds with large string support enabled as well.  I feel like the defaults for `build.sh` and invoking `cmake` directly should be consistent.

Reply:
Ok, that makes sense. I will change it.

Reply:
Changed the compile flag to `CUDF_LARGE_STRINGS_DISABLED` so if not specified, large strings is enabled.